src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStringToArray = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
+  const sourceStringToObject = Object.fromEntries(
+    sourceStringToArray.map((item) => {
+      const [key, value] = item.split(':', 2);
+
+      return [key.trim(), value.trim()];
+    }),
```

Comment:
There is an unnecessary comma at the end of this line. It should be removed to avoid syntax errors.